lib/editor/reducers/timetable.js

```diff
@@ -98,7 +97,8 @@ const timetable = (state: TimetableState = defaultState, action: Action): Timeta
           const path = `${action.payload.rowIndexes[i]}.${col.key}`
           if (isTimeFormat(col.type)) {
             const currentVal = objectPath.get(trips, path)
-            const value = currentVal + (action.payload.offset % 86399) // ensure seconds does not exceed 24 hours
+            // Arrival time can be > 24 hours for service day, so do not block offsetting by 24 hours.
+            const value = currentVal + action.payload.offset
```

Comment:
If there are blank cells in the row that is being offset, the blank cells will be filled with the time with the modulo applied if the applicable stop time is after midnight, and a validation error is shown for these cells because the time in these cells is before the time in the cell referenced in this code. Do we want to remove that behavior for those cells too?

Reply:
I think the better behaviour is to ignore cells that are initially blank